utils/appliance.py

```diff
@@ -416,6 +425,22 @@ def version(self):
     def log(self):
         return create_sublogger(self.address)
 
+    def appliance_object(self):
```

Comment:
I bet this is slow as hell...

I've been threatening to make the multithreaded-multiprovider-method-doer prototyped in [scripts/sync_template_tracker.py](https://github.com/RedHatQE/cfme_tests/blob/master/scripts/sync_template_tracker.py#L24) a generic thing that we can use elsewhere, and this probably a good use-case for it.